src/components/OrderForm/OrderForm.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import './OrderForm.css';
+
+function OrderForm({ formData, handleInputChange, handleSubmit }) {
+  return (
+    <form onSubmit={(e) => e.preventDefault()}>
+      <label htmlFor="voornaam">Voornaam</label><br />
+      <input type="text" id="voornaam" name="voornaam" value={formData.voornaam} onChange={handleInputChange} /><br />
+
+      <label htmlFor="achternaam">Achternaam</label><br />
+      <input type="text" id="achternaam" name="achternaam" value={formData.achternaam} onChange={handleInputChange} /><br />
+
+      <label htmlFor="leeftijd">Leeftijd</label><br />
+      <input type="number" id="leeftijd" name="leeftijd" value={formData.leeftijd} onChange={handleInputChange} /><br />
+
+      <label htmlFor="postcode">Postcode</label><br />
+      <input type="text" id="postcode" name="postcode" value={formData.postcode} onChange={handleInputChange} /><br />
+
+      <label htmlFor="frequency">Hoe vaak wil je bestellen?</label><br />
+      <select id="frequency" name="frequency" value={formData.frequency} onChange={handleInputChange}>
+        <option value="weekly">Iedere week</option>
+        <option value="biweekly">Om de week</option>
+        <option value="monthly">Iedere maand</option>
+      </select><br />
```

Comment:
Het gebruik van `<br />` is niet heel netjes gebruik hier margin of padding voor!